ansible/roles/installation/tasks/main.yml

```diff
@@ -56,23 +56,23 @@
     THREE_SIXTY_DIALOG_PARTNER_ID: "{{ rails.three_sixty_dialog.partner.id | default('') }}"
     THREE_SIXTY_DIALOG_PARTNER_USERNAME: "{{ rails.three_sixty_dialog.partner.username | default('') }}"
     THREE_SIXTY_DIALOG_PARTNER_PASSWORD: "{{ rails.three_sixty_dialog.partner.password | default('') }}"
-
-  community.general.docker_compose:
+  community.docker.docker_compose_v2:
     project_src: /home/ansible
-    build: no
-    debug: yes
-    pull: yes
-    restarted: yes
+    recreate: always
+    build: never
+    pull: always
+    state: present
+    wait: true
```

Comment:
@mattwr18 this is a nice new feature that will ensure that all containers are healthy before proceeding. In our pair-programming today we were facing the issue of silent failures.